where_is_my_sddm_theme/example_configs/verbose.conf

```diff
@@ -0,0 +1,45 @@
+[General]
+# Password mask character
+passwordCharacter=*
+# Mask password characters or not ("true" or "false")
+passwordMask=true
+# value "1" is all display width, "0.5" is a half of display width etc.
+passwordInputWidth=0.5
+# Background color of password input
+passwordInputBackground=
+# Radius of password input corners
+passwordInputRadius=
+# "true" for visible cursor, "false"
+passwordInputCursorVisible=true
+# Font size of password (in points)
+passwordFontSize=96
+passwordCursorColor=random
+passwordTextColor=
+
+cursorBlinkAnimation=true
+
+# Show or not sessions choose label
+showSessionsByDefault=true
+# Font size of sessions choose label (in points).
+sessionsFontSize=24
+
+# Show or not users choose label
+showUsersByDefault=true
+# Font size of users choose label (in points)
+usersFontSize=48
+
+# Path to background image
+background=
+# Or use just one color
+backgroundFill=#000000
+# Fill mode for image background
+# Value must be on of: aspect, fill, tile, pad
+backgroundFillMode=aspect
+
+# Default text color for all labels
+basicTextColor=#ffffff
+
+# Radius of background blur
+blurRadius=
+
+showHelp=true
```

Comment:
Should add this setting to all other configs, because all config options (even if they are the same with default value) should be explicitly specified

Reply:
It should be `showHelp=false` in all other configs